test/integration/parachain/staging/vaults.test.ts

```diff
@@ -191,16 +195,29 @@ describe("vaultsAPI", () => {
 
     it("should getFees", async () => {
         const vault1Id = newAccountId(api, vault_1.address);
-        const feesWrapped = await interBtcAPI.vaults.getFeesWrapped(
+        const feesWrapped = await interBtcAPI.vaults.getWrappedReward(
             vault1Id,
             currencyIdToLiteral(vault_1_id.currencies.collateral) as CollateralIdLiteral,
             currencyIdToLiteral(vault_1_id.currencies.wrapped) as WrappedIdLiteral
         );
-        assert.isTrue(feesWrapped.gte(InterBtcAmount.zero));
+        assert.isTrue(feesWrapped.gt(InterBtcAmount.zero));
+
+        const kintReward = await interBtcAPI.vaults.computeReward(
+            vault1Id,
+            vault1Id,
+            currencyIdToLiteral(vault_1_id.currencies.collateral) as CollateralIdLiteral,
+            CurrencyIdLiteral.INTR
+        );
+        console.log(`kint rwad: ${kintReward.toString()}`);
+        console.log(`wrapped rwd: ${feesWrapped}`);
```

Comment:
Intentional or accidentally left in?

Reply:
Accidental, but since it's just a test I'll remove in the next PR to avoid re-running